### PR TITLE
Saving private key with ".pem" extension.

### LINKED ISSUE #4: Downloaded key should have ".pem" extension
The downloaded key file should have the ".pem" extension.

### PR DESCRIPTION
**Pull request by wcmjunior**
## what
* Add `.pem` extension to private key file

## why
* Fixes #4.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -34,6 +34,6 @@ resource "null_resource" "save_ssh_keys" {
   }
 
   provisioner "local-exec" {
-    command = "echo \"${tls_private_key.default.private_key_pem}\" > ${var.ssh_public_key_path}/${module.label.id} && chmod 600 ${var.ssh_public_key_path}/${module.label.id}"
+    command = "echo \"${tls_private_key.default.private_key_pem}\" > ${var.ssh_public_key_path}/${module.label.id}.pem && chmod 600 ${var.ssh_public_key_path}/${module.label.id}.pem"
```
Comment (osterman) [inline main.tf]: I propose we add a parameter called `private_key_extension` and default it to `""` for backwards compatibility. I think that satisfies both requirements. @comeanother @SweetOps any thoughts?

Comment (osterman) [reply main.tf]: e.g. I've used the `id_rsa` extension in the past.

Comment (wcmjunior) [reply main.tf]: Great. I'll create the parameter and update the pull request. Thanks.

**Comment by osterman** [comment]
We're incorporating your change in a new PR: https://github.com/cloudposse/terraform-aws-key-pair/pull/7

**Comment by wcmjunior** [comment]
Great, @osterman. I couldn't implement those suggested changes. Sorry. Your solution is better yet. Regards.

**Pull request closed without merging**